lib/api_client/connection/middlewares/request/logger.rb

```diff
@@ -47,6 +47,15 @@ def gather_response_debug_lines(response, taken_sec, debug_lines)
   end
 
   def current_stamp_millisec
-    Process.clock_gettime(Process::CLOCK_MONOTONIC, :millisecond)
+    version = RUBY_VERSION.split('.').map(&:to_i)
```

Comment:
Now you're run this check every for every method call. I would extract this and choose implementation on load time, sth like:
```
class CurrentTimestamp
   class << self
      if RUBY_VERSION =~ ...
         def milis
         end
     else
        def milis
        end
   end
end
```